### PR TITLE
[jsk_fetch_startup] Shutdown robot if go-to-kitchen timeout or fail

### PR DESCRIPTION
**Pull request by 708yamaguchi**
~Please merge after https://github.com/jsk-ros-pkg/jsk_robot/pull/1452~

From https://github.com/knorth55/jsk_robot/pull/170

The other day, fetch is stucked during AUTO-DOCK after INSPECT-TRASHCAN.

```
Following smach is reported.
 - At 2021-12-25T09:00:22, Active states is REPORT-START-GO-TO-KITCHEN.
 - At 2021-12-25T09:00:25, Active states is GET-LIGHT-ON.
 - At 2021-12-25T09:00:26, Active states is ROOM-LIGHT-ON.
 - At 2021-12-25T09:00:32, Active states is MOVE-TO-SINK-FRONT.
 - At 2021-12-25T09:01:07, Active states is INSPECT-KITCHEN.
 - At 2021-12-25T09:01:26, Active states is MOVE-TO-TRASHCAN-FRONT.
 - At 2021-12-25T09:01:35, Active states is INSPECT-TRASHCAN.
```

From a battery protection point of view, we want fetch to shutdown in this situation.

With PR, fetch shuts down itself when fetch detects timeout or failure in `go-to-kitchen` demo.

`timeout` is useful when robot stuck by navigation problem. (e.g. robot gets too close to the wall and cannot move anymore)
`failure` is useful when robot stuck by program problem. (e.g. go-to-kitchen main roseus program stops by program bugs.)

### CODE REVIEW

File: jsk_fetch_robot/jsk_fetch.rosinstall.melodic [hunk 1]
```diff
@@ -58,10 +60,12 @@
     local-name: jsk-ros-pkg/jsk_robot
     uri: https://github.com/knorth55/jsk_robot.git
     version: fetch15
+# Use https://github.com/knorth55/app_manager_utils master branch
+# after https://github.com/knorth55/app_manager_utils/pull/40 is merged
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch.rosinstall.melodic:64]: https://github.com/knorth55/app_manager_utils/pull/40
it is already merged

**Review by knorth55 — CHANGES_REQUESTED** [review]

**Comment by knorth55** [comment]
this PR depends on https://github.com/PR2/app_manager/pull/49.
Waiting for upstream.

File: jsk_fetch_robot/jsk_fetch.rosinstall.melodic (hunk 1 above)
Comment (708yamaguchi) [reply jsk_fetch_robot/jsk_fetch.rosinstall.melodic]: I rebased https://github.com/jsk-ros-pkg/jsk_robot/pull/1452, in which this rosinstall problem is resolved.

**Comment by 708yamaguchi** [comment]
@knorth55

> this PR depends on https://github.com/PR2/app_manager/pull/49.
> Waiting for upstream.

I rebased https://github.com/jsk-ros-pkg/jsk_robot/pull/1452, in which this rosinstall problem is resolved.

**Review by knorth55 — APPROVED** [review]

**Pull request merged**